CHANGELOG.next.md

```diff
@@ -60,7 +60,7 @@ Thanks, you're awesome :-) -->
 * Include formatting guidance and examples for MAC address fields. #456
 * New section in ECS detailing event categorization fields usage. #1242
 * `user.changes.*`, `user.effective.*`, and `user.target.*` field reuses are GA. #1271
-* Bump jinja2 from 2.11.2 to 2.11.3 #1310
+* Update Python dependencies #1310, #1318
```

Comment:
Opted to consolidate the two Python dependency updates into a single changelog entry.